### ISSUE TITLE
The estimatedMetadata is wrong

### ISSUE THREAD

**Issue by Gustry** [3liz/qgis-pgmetadata-plugin#27]
I thought @mdouchin could fix it with #26 but for me it still wrong.
I tried to disable in the QGS project, but still.

Either : 
* Use the show feature count, convienent for the warnings about orphan tables
* in the attribute table

We need to investigate in this.

**Comment by Gustry** [comment]
QGIS is not using the result of the VACUUM to get the estimated feature count. Since QGIS 3.10, QGIS is using `EXPLAIN` to get the feature count.

The `Plan Rows` makes quite different results and is not following the `VACUUM ANALYSE` that people might run on their database.

Look this example:

```sql
EXPLAIN (FORMAT JSON) SELECT 1 FROM pgmetadata.contact
-- QUERY PLAN [{'Plan': {'Node Type': 'Seq Scan', 'Parallel Aware': False, 'Relation Name': 'contact', 'Alias': 'contact', 'Startup Cost': 0.0, 'Total Cost': 15.1, 'Plan Rows': 510, 'Plan Width': 4}}]
-- > 'Plan Rows': 510

VACUUM ANALYSE pgmetadata.contact;
SELECT reltuples as approximate_row_count  FROM pg_class WHERE oid = 'pgmetadata.contact'::regclass;
-- 0.0

SELECT COUNT(*) FROM pgmetadata.contact;
-- 0

EXPLAIN (FORMAT JSON) SELECT 1 FROM pgmetadata.contact;
-- Still 510
```

**Comment by Gustry** [comment]
Some fixes has been merged for 3.18.0 and 3.16.3

I think we should close this ticket.
